frontend/src/components/profProfile/AvailabilityCard.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+
+interface AvailabilityCardProps {
+  day: string;
+  // Assuming comma separated time ranges e.g. "8am-10am, 2pm-4pm"
+  times: string;
+}
+
+const AvailabilityCard = ({ day, times }: AvailabilityCardProps) => {
+  const timeList = times.split(',').map(time => time.trim());
+
+  const isAvailable = timeList.length > 0 && timeList[0] !== '';
+  //TODO:
+  //      Scale for more times
+  //      Add hover animation 
+
+  // Conditional class names for styling based on availability
+  const borderColorClass = isAvailable ? 'border-light-blue hover:bg-gray-200 transition cursor-pointer' : 'border-opacity-95 border-blue-200';
+  const textColorClass = isAvailable ? 'text-okb-blue' : 'text-opacity-95 text-blue-200';
+
+  return (
+    <div className={`min-h-[8rem] w-32 p-4 border-2 rounded-[20px] space-y-2 flex flex-col ${borderColorClass}`}>
```

Comment:
The webpage looks great! Nice job implementing all of the components. When adjusting the width of the webpage, the boxes pertaining to the dates and appointment times remain the same size, which causes some overlap (see screenshot). Perhaps add some CSS to each box so that it adjusts according to the width of the whole webpage would look more aesthetically pleasing. Other than that, it looks great!
<img width="915" alt="Screen Shot 2023-10-17 at 6 50 41 PM" src="https://github.com/cornellh4i/okb-hope/assets/123703741/eea1f95f-3c36-40ad-990d-08795c80ea6a">